gremlin-dotnet/build/generate.groovy

```diff
@@ -175,6 +137,11 @@ radishGremlinFile.withWriter('UTF-8') { Writer writer ->
             '            list.RemoveAt(0);\n' +
             '            return f.Invoke(g, parameters);\n' +
             '        }\n' +
+            '\n' +
+            '        public static bool HasTraversal(string scenarioName)\n' +
```

Comment:
where is `HasTraversal` used?

Reply:
it was being used, but i switched to statically defining the unsupported lambda tests. removing it.